### PR TITLE
add Spiritsmc

### PR DESCRIPTION
**Pull request by not16hir0**
## Type of change

- [x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by scrummer** [comment]
@not16hir0 Can you (at least) compress your background images? They're reeeally big

**Comment by niklaswa** [comment]
Hey there, I think, it would be better if you'd remove the logo from the background image as it will get overlayed at the certain places automatically (for example on laby.net)

File: minecraft_servers/Spiritsmc/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+   "nice_name":"SpiritsMC",
+   "direct_ip":"spiritsmc.com",
+   "supported_languages":[
+      "ar",
+      "en"
+   ],
+   "social":{
+      "web_shop":"https://store.spiritsmc.com",
+      "youtube":"https://www.youtube.com/channel/UCt0j6W900KLGz7AQV2bh0bA",
+      "discord":"https://discord.gg/spr"
```
Comment (JakobBruening) [inline minecraft_servers/Spiritsmc/manifest.json:12]: Bildschirmfoto 2022-09-20 um 14 50 03

**Review by JakobBruening — CHANGES_REQUESTED** [review]

**Comment by scrummer** [comment]
@not16hir0 Please fix the open issues within the next days. Otherwise we close the PR to keep it all clean.

**Comment by JakobBruening** [comment]
I will close the PR due to inactivity.
Please re-open or re-create as soon as the comments are resolved.

**Pull request closed without merging**